source/includes/_invoices.md

```diff
@@ -2,22 +2,39 @@
 
 Invoices allow you to store invoice information for later payment.
 
+The Invoice integrates a system of **Invoice Line**. An invoice line is defined by an `amount`, a `currency` and a `description`.
```

Comment:
This is not the best English... 

Maybe better with: The invoice has invoice lines (Maybe a link to the description here). An invoice line consists mainly of `amount`, a `currency` and a `description`. See description for more details.